flocx_market/api/app.py

```diff
@@ -1,20 +1,16 @@
-import pecan
-from pecan import make_app
-from flocx_market import model
-from flocx_market.api import config as api_config
+import os
+from flask import Flask
+from flask import Blueprint
 
+my_bp = Blueprint("my_bp", __name__)
 
-def get_pecan_config():
-    filename = api_config.__file__.replace('.pyc', '.py')   # get the absolute path of the pecan config.py
-    return pecan.configuration.conf_from_file(filename)
 
+@my_bp.route('/')
+def index():
+    return "Hello World"
```

Comment:
Most OpenStack services serve something at the index; for example, if you curl your keystone endpoint, you'll see something like this:

```
[stack@localhost controllers(keystone_admin)]$ curl http://192.168.1.2:5000
{"versions": {"values": [{"status": "stable", "updated": "2019-01-22T00:00:00Z", "media-types": [{"base": "application/json", "type": "application/vnd.openstack.identity-v3+json"}], "id": "v3.12", "links": [{"href": "http://192.168.1.2:5000/v3/", "rel": "self"}]}]}}
```

I'd update this to something similar.